bundles/pim/src/main/java/com/adobe/cq/commerce/demandware/pim/impl/DemandwareProductImporter.java

```diff
@@ -132,14 +132,18 @@ protected boolean validateInput(SlingHttpServletRequest request, SlingHttpServle
 
         String storePath = request.getParameter("storePath");
         String storeName = request.getParameter("storeName");
-        if (StringUtils.isEmpty(storePath) || StringUtils.isEmpty(storeName)) {
+        instanceId = request.getParameter("instanceId");
+        if (StringUtils.isEmpty(storePath) || StringUtils.isEmpty(storeName) || StringUtils.isEmpty(instanceId)) {
```

Comment:
StringUtils.isEmpty(instanceId) is checked also in line 140, that will never be reached